### PR TITLE
Add roadmap for v1.0

### PR DESCRIPTION
**Pull request by zhouqiang-cl**
### What is changed and how does it work?
This pr add a roadmap for chaos mesh
### Check List 

Tests 
 - No code

Code changes

 - No Code changes

Side effects

 - No effects on code

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
issue https://github.com/pingcap/chaos-mesh/issues/124

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Chaos Mesh Roadmap
+
+This document defines the roadmap for Chaos Mesh development.
+
+## v0.1.0
+- [ ] Support bpf injection for time skew chaos
+- [ ] Add container kill/failure chaos
+- [ ] Add cpu chaos
+- [ ] Add memory chaos
```
Comment (cwen0) [inline ROADMAP.md]: Make scheduler optional？

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Chaos Mesh Roadmap
+
+This document defines the roadmap for Chaos Mesh development.
+
+## v0.1.0
```
Comment (cwen0) [inline ROADMAP.md]: not v0.1.0?

Comment (dcalvin) [reply ROADMAP.md]: +1

File: ROADMAP.md (hunk 1 above)
Comment (mahjonp) [reply ROADMAP.md]: What's the difference between pod failure and container failure?

File: ROADMAP.md (hunk 2 above)
Comment (zhouqiang-cl) [reply ROADMAP.md]: how about your suggest

File: ROADMAP.md (hunk 1 above)
Comment (mahjonp) [reply ROADMAP.md]: can we provides more detailed information about each chaos type and let community know the detail of each item.

File: ROADMAP.md (hunk 2 above)
Comment (dcalvin) [reply ROADMAP.md]: Checked some resources, I think it should be fine. Before 1.0, we can call it V0.X

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+- [ ] Support bpf injection for time skew chaos. Simulate time jump forward or backword.
+- [ ] Add container kill/failure chaos. Support kill the special container in multi containers pod.
+- [ ] Add cpu chaos. Simulate cpu is busy.
+- [ ] Add memory chaos. Simulate can not alloc memory.
```
Comment (Yisaer) [inline ROADMAP.md]: cpu and memory chaos seems like `node` level chaos. Maybe we could support and extend more abilities and methods of selector for existed chaos first.

Comment (zhouqiang-cl) [reply ROADMAP.md]: How about "Make scheduler optional"

Comment (zhouqiang-cl) [reply ROADMAP.md]: How about "Make scheduler optional"

Comment (Yisaer) [reply ROADMAP.md]: @zhouqiang-cl In my view, `Make scheduler optional` is necessary for v1.0 which would make `chaos api` both user-friendly and developer-friendly.

Comment (zhouqiang-cl) [reply ROADMAP.md]: Yes. already add it below

Comment (zhouqiang-cl) [reply ROADMAP.md]: what do you mean about `selector`, did it means select of pod? seems we have already support it

File: ROADMAP.md (hunk 1 above)
Comment (zhouqiang-cl) [reply ROADMAP.md]: we will add detail info in issues

**Pull request merged**